alns/solve_options.go

```diff
@@ -0,0 +1,18 @@
+package alns
+
+// NewSolveOptions returns a new instance of SolveOptions. The default
+// options are:
+//   - iterations: -1
+//   - solution_buffer_size: 100
+func NewSolveOptions() SolveOptions {
+	return SolveOptions{
+		Iterations:         -1,
+		SolutionBufferSize: 100,
+	}
+}
+
+// SolveOptions contains the options for the solve process.
+type SolveOptions struct {
+	Iterations         int `json:"iterations" default:"-1"`
+	SolutionBufferSize int `json:"solution_buffer_size" default:"100"`
```

Comment:
What is the buffer size doing again, we should document now we are exposing it

Reply:
It configures the length of the solution channel, but instead of describing it, I decided to remove and configure it internally as I don't see that this needs to be configurable at this time. If we find out otherwise, we can re-introduce the field.